src/lib/connection/connector/fallback-handlers.js

```diff
@@ -96,13 +96,13 @@ export const openLink = url => {
 };
 
 /**
- * Open an external link
+ * Handler for unpossible functions
  *
- * @param {String} url address
+ * @param {String} cmd command name
  */
-export const nativeUpload = url => {
```

Comment:
Why have you changed nativeUpload?
That should be not gone.

Reply:
Look downward. I did a unsupported function and merged the two cases. After looking through the code, I remembered, that we decided to end always in the fallback and throw warnings for features which are not doable with fallbacks.